core-plugins/src/main/java/io/cdap/plugin/common/FileSetUtil.java

```diff
@@ -42,9 +42,6 @@ public static void configureParquetFileSet(String configuredSchema, FileSetPrope
     String hiveSchema = parseHiveSchema(configuredSchema, configuredSchema);
```

Comment:
hiveSchema is not used anymore

Reply:
Also the method does not have anything about "parquet" anymore, it just calls `properties.add`. Consider inlining

Reply:
Noted. I will resolve hive related refactorings and deletions in a separate PR.